src/themis/secure_comparator.c

```diff
@@ -38,7 +38,7 @@ static themis_status_t ed_sign(uint8_t pos, const uint8_t *scalar, uint8_t *sign
 {
 	uint8_t r[ED25519_GE_LENGTH];
 	ge_p3 R;
-	uint8_t k[64];
+	uint8_t k[64]; //temp buffer for group elements and hash
```

Comment:
The typical use case here would be that if you want to verify the code - you'll take the crypto paper, which defines the algorithm. And if the paper says `k`, but you have `temp_buff` - this is confusing as well.